<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnrolementScreen</class>
 <widget class="QDialog" name="EnrolementScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enrollement</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>601</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="enrscurrentQuality">
      <property name="text">
       <string>Current Quality : xxxxxxxxxxxxxxxxxxxxxxxx</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="enrsenrollNew">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Enroll New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enrsstartBtn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>280</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enrsmenuBtn">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>580</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>321</width>
     <height>241</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>319</width>
      <height>239</height>
     </rect>
    </property>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
